guides/getting-started/set-up/anoncreds-rs.md

```diff
@@ -0,0 +1,126 @@
+# Anoncreds-rs
+
+[AnonCreds](https://github.com/hyperledger/anoncreds) is a direct
+implementation of the [AnonCreds V1.0
+specification](https://hyperledger.github.io/anoncreds-spec/) that provides
+functionality like; creating a schema object, creating a credential definition
+object, creating a credential, verifying a proof presentation and much more.
+
+:::caution
+
+Support for AnonCreds in Aries Framework JavaScript is currently experimental.
+We recommend new projects to use AnonCreds from the start, and also to migrate
+existing projects to Anoncreds, from the indy-sdk. However, projects may
+experience some issues. If you encounter any issues, please [open an
+issue](https://github.com/hyperledger/aries-framework-javascript/issues/new).
+
+Currently, there are few limitations to using AnonCreds.
+
+- React Native projects leveraging [Expo](https://expo.dev) are not supported
+  at the moment. Support for Expo will be added soon.
+- When running in Node.JS, only Node.JS 18 is suported for now. See [Supported
+  Node.JS versions for AnonCreds](#supported-nodejs-versions-for-anoncreds)
+
+:::
+
+### Installing AnonCreds
+
+When using Aries Framework JavaScript with AnonCreds, there are a few extra
+dependencies that need to be installed. We need to install the
+`@hyperledger/anoncreds`, which contains the interfaces, and
+`@hyperledger/anoncreds-rs` which is an implementation which depends on a
+wrapper of anoncreds-rs. Secondly, we need to add native bindings for the
+specific platform `@hyperledger/anoncreds-<platform>`. Currently there are
```

Comment:
Can you keep the installation as consistent as possible? So if modifying this by adding the `nodejs` etc.., we should add it to the other existing setup guides as well.